.pre-commit-config.yaml

```diff
@@ -0,0 +1,146 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: "9260cbc9c84c06022993bfbcc42fdbf0305c5b8e"
+    hooks:
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: name-tests-test
+        args: ["--pytest-test-first"]
+      - id: trailing-whitespace
+
+  # - repo: https://github.com/hakancelik96/unimport
```

Comment:
They can be removed (in fact I will remove them). Should you wish to add additional hooks later e.g. because you work with some special file types etc. you can find a list [here](https://pre-commit.com/hooks.html).